lightly/api/api_workflow_datasources.py

```diff
@@ -26,17 +30,27 @@ def download_raw_samples(self, from_: int = 0, to: int = None):
             to=to,
         )
         cursor = response.cursor
-        data = _convert_raw_data_to_list(response.data)
+        samples = response.data
         while response.has_more:
             response: DatasourceRawSamplesData = self._datasources_api.get_list_of_raw_samples_from_datasource_by_dataset_id(
                 dataset_id=self.dataset_id, cursor=cursor
             )
             cursor = response.cursor
-            new_data = _convert_raw_data_to_list(response.data)
-            data.extend(new_data)
-        return data
+            samples.extend(response.data)
+        samples = [(s.file_name, s.read_url) for s in samples]
+        return samples
 
-    def download_new_raw_samples(self):
+    def download_new_raw_samples(self) -> List[Tuple[str, str]]:
+        """Downloads filenames and read urls of unprocessed samples from the datasource.
+
+        All samples after the timestamp of `ApiWorkflowClient.get_processed_until_timestamp()` are 
+        fetched. After downloading the samples the timestamp is updated to the current time.
+        This function can be repeatedly called to retrieve new samples from the datasource.
+        
+        Returns:
+            Samples (list(tuple)): Each tuple contains (filename, url) of the respective sample
```

Comment:
Don't include the type in the docstring. It is sufficient to have it in the declaration.